### ISSUE TITLE
failed to connect redis - WRONGPASS invalid username-password pair

### ISSUE THREAD

**Issue by Densol92** [toluaina/pg-sync#14]
PGSync version: 
1.1
Postgres version: 
12
Elasticsearch version: 
7.6.2
Redis version: 

Python version: 
3.7
Problem Description: 
docker-compose airbnb example is not working`

Error Message (if any): 

```
Traceback (most recent call last):
  File "examples/airbnb/schema.py", line 130, in <module>
    main()
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "examples/airbnb/schema.py", line 125, in main
    teardown(config=config)
  File "pgsync/helper.py", line 44, in pgsync.helper.teardown
  File "pgsync/redisqueue.py", line 33, in pgsync.redisqueue.RedisQueue.__init__
  File "/usr/local/lib/python3.7/site-packages/redis/client.py", line 1351, in ping
    return self.execute_command('PING')
  File "/usr/local/lib/python3.7/site-packages/redis/client.py", line 875, in execute_command
    conn = self.connection or pool.get_connection(command_name, **options)
  File "/usr/local/lib/python3.7/site-packages/redis/connection.py", line 1185, in get_connection
    connection.connect()
  File "/usr/local/lib/python3.7/site-packages/redis/connection.py", line 561, in connect
    self.on_connect()
  File "/usr/local/lib/python3.7/site-packages/redis/connection.py", line 637, in on_connect
    auth_response = self.read_response()
  File "/usr/local/lib/python3.7/site-packages/redis/connection.py", line 752, in read_response
    raise response
redis.exceptions.ResponseError: WRONGPASS invalid username-password pair


```

**Comment by toluaina** [comment]
This has now been resolved in release 1.1.1

**Comment by toluaina** [comment]
closing issue. fixed in 1.1.1
